recurly-android-sdk/.gitignore

```diff
@@ -4,6 +4,7 @@
 /.idea/libraries
 /.idea/tasks.xml
 /.idea/gradle.xml
+/.idea/misc.xml
```

Comment:
I'm a little bit unsure why this whole folder isn't ignored. This should be safe though.